www/main/navigator/search/components/SearchContent.jsx

```diff
@@ -228,7 +229,7 @@ const SearchContent = () => {
           />
         </div>
       </div>
-      <div className="search-results">
+      <div className={classNames('search-results', isShowFiltersTag && 'filter-applied')}>
```

Comment:
@theamanjs ji, why do we need a function call here?

with function:
````
<div className={classNames('search-results', isShowFiltersTag && 'filter-applied')}>
````

Whereas, without function we can simply use the template string directly
````
<div className=`'search-results' ${isShowFiltersTag && 'filter-applied'}`>
````

Reply:
veerji classnames is a utility function that I created to avoid multiple conditions on an element and to avoid a major issue that comes while using the second approach you mentioned.

when we will use `isShowFilersTag && 'filters-applied'` and the condition is false it's gonna render a class named `false` and we don't need that. To avoid that false we will have to use an unwanted ternary operator.

Reply:
Got it. Good to go.